### ISSUE TITLE
Screen orientation regression when sharing file (image) - last 2 or three versions

### ISSUE THREAD

**Issue by cyclomani** [mollyim/mollyim-android#330]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Hello,

How to repeat:
Turn device auto-rotatation on (if not already).
View a picture in landscape mode and select the 'share' action.

Up to the last two or three versions, molly respected the landscape mode and I could type a message (in landscape mode) before sending the attachment.
Since two or three versions, molly (and only molly) rotates to portrait mode as soon as I selected the recipient of the 'share' dialog, before I can send the attachment.

That's just annoying, not a serious bug of course, but I'd very much appreciate if usability isn't made worse for no reason.

### Steps to reproduce

1. Turn device auto-rotatation on (if not already).
2. View a picture in landscape mode and select the 'share' action.
3. Select one ore more recipients if not already selected from the 'share' menu

### Molly version

v7.8.1-1-FOSS

### Android version

Android 13

### Device

Google Pixel 7

**Comment by valldrac** [comment]
> Up to the last two or three versions, molly respected the landscape mode and I could type a message (in landscape mode) before sending the attachment.

Thanks for the report. I assume you tested the same version in Signal and didn't have this issue. Can you confirm that?

**Comment by cyclomani** [comment]
> > Up to the last two or three versions, molly respected the landscape mode and I could type a message (in landscape mode) before sending the attachment.
> 
> Thanks for the report. I assume you tested the same version in Signal and didn't have this issue. Can you confirm that?

Nope, no Signal app for testing on the same device, but it seems this problem was introduced with Signal 7.6.2 - at least on another device.
Are molly users supposed to report only molly-changeset specific errors?  I know several other projects don't give anything about upstream bugs, but from a user point of view, I don't care much about upstream because I'm not running upstream project but the app in question.  Just out of curiosity, I can understand if molly doesn't have resources to take care about anythung but their own modifications.
Thanks!

**Comment by valldrac** [comment]
We appreciate all bug reports from users. But the first step for us is to determine where the bug originated.

This step is important because fixing bugs depends a lot on where they're from, whether it's upstream or because of changes in Molly.

We give priority to fixing bugs specific to Molly rather than those from upstream. There are many reasons for this, not just because we're short on resources. The less collaboration and coordination there's between upstream project and the fork, the better it's for each to handle their own bugs.

**Comment by godovski** [comment]
I filed a bug on this just now upstream, since I could not find any report of this issue and it annoys me as well: https://github.com/signalapp/Signal-Android/issues/14737
